aptos/contracts/sources/pyth.move

```diff
@@ -179,6 +179,12 @@ module pyth::pyth {
         update_cache(batch_price_attestation::destroy(batch_price_attestation::deserialize(vaa::destroy(vaa))));
     }
 
+    /// Allow anyone to update the cache with given updates. For testing purpose only.
+    #[test_only]
+    public fun update_cache_test(updates: vector<PriceUpdate>) {
+        update_cache(updates);
+    }
```

Comment:
Thank-you! Would it be possible to move this to the `pyth::pyth_test` module?

Reply:
Yeah, that is possible and better. Will move to `pyth::pyth_test` module.